data/data.js

```diff
@@ -7,39 +7,33 @@ export const data = {
     community: 'https://forms.gle/hBXbTV7DYBsRXiQTA',
     sponsors: 'https://forms.gle/MnJNbzU4K5WohuhN9',
   },
-  about: {
-    title: 'About us',
-    content:
-      'We are the first React conference for Spanish speakers in Latin America. This is an event by the community to the community! Come and find the latest knowledge and awesome experiences around React in Medellín, Colombia.',
-  },
   numbers: {
     title: 'The conference in numbers',
     fineprint: 'July 18 - Conference',
     content: [
-      <>
-        <span>300+</span>
-        <span>people</span>
-      </>,
-      <>
-        <span>1</span> <span>track</span>
-      </>,
-      <>
-        <span>11</span> <span>speakers</span>
-      </>,
+      {
+        value: 'number_of_people_value',
+        label: 'number_of_people_label',
+      },
+      {
+        value: 'number_of_tracks_value',
+        label: 'number_of_tracks_label',
+      },
+      {
+        value: 'number_of_speakers_value',
+        label: 'number_of_speakers_label',
+      },
     ],
   },
   tickets: {
     title: 'Tickets 🎟',
-    content:
-      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ullamcorper neque nunc mattis id gravida porttitor.',
+    content: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ullamcorper neque nunc mattis id gravida porttitor.',
```

Comment:
We're not using this at all, so, I think we could either:

- Add a tickets section (that was the original intent of this), but we ended up redirecting the user to ti.to
- Remove this section once and for all

Thoughts, @xkill-9 @alejogs4?

Reply:
I think that to have tickets sections it would be useful for users, it would work as a reference for possible attendees to know past prices, for example right now if someone visits our webpage will not see the last batch price. What do you think?

Reply:
I like the idea @alejogs4!

One caveat, we don't have designs for for the tickets section just yet. Let's discuss this in Twist and iterate on it afterwards to prevent our scope from growing. What do you think?

Reply:
Yeah I thought it as well. I agree

Reply:
That's a good point @alejogs4